versions/1/docs/spec.yml

```diff
@@ -0,0 +1,8 @@
+  spec:
+    additionalContents: true
```

Comment:
Are we expecting to see docs other than `README.md` in the docs folder? Looking through the `integrations` packages, I'm only seeing `README.md` files under docs. I'd prefer that we are stricter now by setting `additionalContents: false` and then loosen the restriction later if/when we have a use case for it.

Reply:
The initial design for the docs directory was that it could contain multiple files which can also reference / include each other but for simplicity we ended up going with just a single file for now. So at the moment, I don't think it should contain any other file so we can lock it down. We can still open it up later if we have new requirements.

Reply:
Fixed!